content/en/monitors/case_management/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Case Management
+kind: documentation
+---
+
```

Comment:
If this is in public beta, would a callout be appropriate? For example:

```suggestion
{{< callout url="#" btn_hidden="true">}}
  The Datadog extension for Visual Studio Code is in public beta. It is intended for Datadog users who have <a href="https://docs.datadoghq.com/synthetics/#pagetitle">Synthetic tests</a> set up already. The extension supports running <a href="https://docs.datadoghq.com/synthetics/api_tests/http_tests">HTTP tests</a> and <a href="https://docs.datadoghq.com/synthetics/browser_tests">browser tests</a>. If the extensions stops working unexpectedly, check for extension updates or <a href=#feedback>reach out to the team</a>.
{{< /callout >}}
```

Reply:
Thanks for the suggestion! I figured the "beta" tag in the UI would be sufficient, but you're right that a callout in the docs may be helpful. I'll ask Cansu and Tanja after the launch about adding one - they're probably pretty busy today.